plugins/stats_table_procedure.cc

```diff
@@ -156,7 +114,51 @@ reconstitute(ML::DB::Store_Reader & store)
 
 
 /*****************************************************************************/
-/* ROLL PROCEDURE                                                            */
+/* STATS TABLE PROCEDURE CONFIG                                              */
+/*****************************************************************************/
+
+DEFINE_STRUCTURE_DESCRIPTION(StatsTableProcedureConfig);
+
+StatsTableProcedureConfigDescription::
+StatsTableProcedureConfigDescription()
```

Comment:
Were you planning to merge the sql expression into a single statement or will we do it latter?